### PR TITLE
Release v1.4.0 actions 

### PR DESCRIPTION
**Pull request by ddimatos**
##### SUMMARY
This pull request addresses issue #432 which is to address the release tasks for the version 1.4.0.

Please see issue #432 for more details as the checklist applies to the changes here.

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

**Comment by ddimatos** [comment]
image
Console log suffix `/view/AC/job/zosAnsible/1326/consoleFull`

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -292,6 +292,38 @@ releases:
...
+        For additional details such as required dependencies and availability review
+
+        the collections `release notes <https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/release_notes.html>`__
+
+        '
+    fragments:
+    - 561-update-directory-create.yml
+    - 566-update-with-symbol-support.yml
+    - v1.4.0_summary.yml
+    release_date: '2022-11-29'
```
Comment (rexemin) [inline changelogs/changelog.yaml:326]: Is this date correct?

**Review by rexemin — APPROVED** [review]
Only had the question I added in `changelog.yaml`, everything else looks good to me.

File: changelogs/changelog.yaml (hunk 1 above)
Comment (ddimatos) [reply changelogs/changelog.yaml]: The date is not correct in terms of the release date, that date in changelog.yaml is not the date that is put in the CHANGLOGE.rst, that CHANGELOG.rst comes from the summary (v1.4.0_summary.yaml). So the date you see in the changelog.yaml  I generated the changelog, but i did find a way to put a future date in the changelog.yaml even though it has no bearing and updated it. `antsibull-changelog release --date 2022-12-07`

**Pull request merged**